openapi.yaml

```diff
@@ -1023,6 +1055,74 @@ components:
                   description: Job reason code
                   example: ABCDE
                   nullable: true
+    DeductionsResource:
+      type: object
+      properties:
+        id:
+          type: string
+          description: OSU ID of a employee
+          pattern: ^\d{9}$
+        type:
+          $ref: '#/components/schemas/DeductionsType'
+        links:
+          $ref: '#/components/schemas/SelfLink'
+        attributes:
+          properties:
+            responseMsg:
```

Comment:
We want a more specific name for any field. What possible "messages" could be returned for this field? If you only care whether it's successful or not, you should be able to tell by the HTTP status code. The field within the body is more of providing some detailed information, e.g. deductions status, deduction types (I don't know they applied, but just as examples), or something like that. In these cases, these fields should be separated and named more descriptively.

Reply:
Some examples:
**ERROR** Could not create MEC PDRBDED deduction base.
**ERROR** 123456789 does not exist in SPRIDEN. Deductions not applied.
**ERROR** Position C1234 Suffix 00 is not an active position for OSU ID 123456789. Deductions not applied.
**ERROR** Could not update TT1 PDRDEDN deduction detail.
SUCCESS

Maybe I call that field something like deduction status instead of response message, and then I can add a field for deduction types applied? Since there are 13, 10 of them could apply correctly, and then it could run into an error on the 11th and return one of those error messages. Open to suggestions.

Reply:
So, It's possible to create multiple deductions but only partially successful? If so, are you considering this as a "successful" call? If there is any error during creation, would you expect the end user to correct something and then try this call again?

Reply:
Will discuss this one during our call.